style.css

```diff
@@ -0,0 +1,415 @@
+  background-position: center;
+  cursor: pointer;
+}
+
+@media (min-width: 992px) {
+  .header {
+    min-height: 92px;
+  }
+
+  .header__head {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+      -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+    justify-content: space-between;
+    min-width: 461px;
+    background: #000000;
+  }
+
+  .header__logo {
+    margin-top: 42px;
+    margin-left: 55px;
+  }
+
+  .header__lang {
+    position: relative;
+    display: block;
+    width: 50px;
+      -ms-flex-item-align: center;
+      -ms-grid-row-align: center;
+    align-self: center;
+    margin-right: 22px;
+    margin-top: 20px;
+  }
+
+  .header__lang-option {
+    position: absolute;
+    width: 31px;
+    height: 25px;
+    left: 388px;
+    top:0;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    text-align: right;
+    text-decoration: none;
+    color: #FFFFFF;
+  }
+
+  .header__lang-option:hover {
+    border-bottom: solid 2px #eeddd0;
+  }
+
+  .header__lang-option::after {
+    position: absolute;
+    top: 3px;
+    left: 45px;
+    display: block;
+    content: '';
+    width: 7px;
+    height: 6px;
+    border: 2px solid #FFFFFF;
+    border-right-color: transparent;
+    border-top-color: transparent;
+      -webkit-transform: rotate(-44deg);
+      -ms-transform: rotate(-44deg);
+    transform: rotate(-44deg);
+  }
+
+  .header__lang-option:hover:after {
+    transform: rotate(135deg);
+  }
+
+  .header__menu {
+    width: 92px;
+    height: 92px;
+  }
+
+  .header__menu:hover {
+    background-color: #eeddd0;
+  }
+
+  .header__burger-line:first-of-type {
+    margin-bottom: 5px;
+  }
+
+  .header__burger-line {
+    width: 35px;
+    height: 4px;
+  }
+
+  .up-section__desription {
+    line-height: 21px;
+    font-size: 15px;
+    font-weight: normal;
+  }
+
+  .up-section__collection-description{
+    width: 475px;
+    margin-left: 45px;
+  }
+
+  .up-section__title {
+    padding-left: 10px;
+    margin-top: 108px;
+    margin-bottom: 2px;
+    width: 475px;
+    height: 164px;
+    font-weight: bold;
+    line-height: 82px;
+    font-size: 80px;
+    }
+
+  .up-section {
+    background: -webkit-linear-gradient(left, #000000 461px, #FFFFFF 460px);
+    background: -o-linear-gradient(left, #000000 461px, #FFFFFF 460px);
+    background: linear-gradient(to right, #000000 461px, #FFFFFF 460px);
+  }
+
+  .up-section__collection {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+        -webkit-box-pack: center;
+        -ms-flex-pack: center;
+    justify-content: center;
+        -webkit-box-align: center;
+        -ms-flex-align: center;
+    align-items: center;
+    padding: 0 40px;
+    }
+
+  .up-section__images {
+    z-index: 10;
+    height: 562px;
+    margin-top: 77px;
+  }
+
+  .up-section__collection-image img {
+    width: 549px;
+    height: 562px;
+        -o-object-position: 50% 50%;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5676737)